docs/content/config/index.md

```diff
@@ -21,9 +21,9 @@ We also provide an `.env.example` file containing a minimal configuration in the
 
 | environment variable     | default   | example                     | description                                                                                                                                            |
 |--------------------------|-----------|-----------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `HD_DOMAIN`              | -         | `https://md.example.com`    | The URL the HedgeDoc instance runs on.                                                                                                                 |
+| `HD_BASE_URL`              | -         | `https://md.example.com`    | The URL the HedgeDoc instance runs on.                                                                                                                 |
 | `PORT`                   | 3000      |                             | The port the HedgeDoc instance runs on.                                                                                                                |
-| `HD_RENDERER_BASE_URL`   | HD_DOMAIN |                             | The URL the renderer runs on. If omitted this will be same as `HD_DOMAIN`.                                                                             |
+| `HD_RENDERER_BASE_URL`   | HD_BASE_URL |                             | The URL the renderer runs on. If omitted this will be same as `HD_BASE_URL`.                                                                             |
```

Comment:
```suggestion
| `HD_RENDERER_BASE_URL`   | `HD_BASE_URL` |                             | The URL the renderer runs on. If omitted this will be same as `HD_BASE_URL`.                                                                             |
```